source/agora/flash/Node.d

```diff
@@ -241,10 +241,8 @@ public abstract class FlashNode : FlashControlAPI
         auto secret = SecretKey(scalar);
         auto address = PublicKey(scalar.toPoint()[]);
         this.managed_keys[address] = secret;
-        Channel[Hash] def;
-        this.channels[address] = def;
-        DList!ChannelConfig def2;
-        this.pending_channels[address] = def2;
+        this.channels[address] = typeof(this.channels[address]).init;
+        this.pending_channels[address] = typeof(this.pending_channels[address]).init;
```

Comment:
Why is it here in the first place ? Does it override something ? Everything is default-initialized in D.

Reply:
Those are complex types, like AA and DList.
These are there so that when you do `this.channels[address].foo()`, you wont throw.